resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/submit/steps/BasicDriverConfigurationStepSuite.scala

```diff
@@ -91,7 +91,7 @@ class BasicDriverConfigurationStepSuite extends SparkFunSuite {
     val resourceRequirements = preparedDriverSpec.driverContainer.getResources
     val requests = resourceRequirements.getRequests.asScala
     assert(requests("cpu").getAmount === "2")
-    assert(requests("memory").getAmount === "256Mi")
+    assert(requests("memory").getAmount === "456Mi")
```

Comment:
I'm wondering - shouldn't this be 256M + 384M?
The documentation says - `executorMemory * 0.10, with minimum of 384`.

Reply:
nvm - @liyinan926 just pointed out that it's being set explicitly above which I totally missed.